aws/sdk/integration-tests/s3/Cargo.toml

```diff
@@ -8,18 +8,20 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dev-dependencies]
+aws-config = { path = "../../build/aws-sdk/sdk/aws-config" }
 aws-http = { path = "../../build/aws-sdk/sdk/aws-http" }
 aws-hyper = { path = "../../build/aws-sdk/sdk/aws-hyper" }
 aws-sdk-s3 = { path = "../../build/aws-sdk/sdk/s3" }
+aws-sdk-sts = { path = "../../build/aws-sdk/sdk/sts" }
```

Comment:
Unused?

Reply:
I don't know exactly why but if I don't include this, it errors out when trying to build and run the test because it can't find aws_smithy_client. Once I added this, the issue went away.